test/test_restful.py

```diff
@@ -147,7 +147,7 @@ def setUpModule():
                 print("Found dbus service:", str(service))
                 try:
                     print("Restarting tpm2-abrmd.service.")
-                    job = manager.RestartUnit('tpm2-abrmd.service', 'fail')
+                    _ = manager.RestartUnit('tpm2-abrmd.service', 'fail')
```

Comment:
Since the return value is not used, we can just skip the value assignment, does pylint complains too?

Reply:
It would not complain, so removed it entirely.